src/puppetlabs/puppetdb/query/reports.clj

```diff
@@ -17,6 +18,7 @@
    :puppet_version String
    :report_format s/Int
    :configuration_version String
+   :metrics (s/maybe s/Any)
```

Comment:
Is this because it could be either a PGObject or a string? Can we validate for that?